src/app/cluster/list/external-cluster/template.html

```diff
@@ -69,47 +69,52 @@
             *matCellDef="let element"
             [attr.id]="'km-clusters-' + element.name">
           <div fxLayoutAlign=" center"
-               fxLayoutGap="8px">{{element.name}}
+               fxLayoutGap="8px">
+            <span>{{element.name}}</span>
+            <span *ngIf="element?.labels['is-imported'] === 'true'"
+                  class="km-label-primary">Imported</span>
           </div>
         </td>
       </ng-container>
 
-      <ng-container matColumnDef="labels">
+      <ng-container [matColumnDef]="Column.Provider">
         <th mat-header-cell
             *matHeaderCellDef
-            class="km-header-cell p-30">Labels
+            class="km-header-cell p-20">Provider
         </th>
         <td mat-cell
             *matCellDef="let element">
-          <km-labels [labels]="element.labels"
-                     [limit]="2"></km-labels>
+          <span [ngClass]="getProvider(element.cloud) === Provider.Custom ?
+              'km-provider-logo km-provider-logo-custom-external-cluster' : 'km-provider-logo km-provider-logo-'+getProvider(element.cloud)"></span>
```

Comment:
`km-provider-logo` can just be moved into `class` instead of `ngClass`

Reply:
will be fixed in sep PR due to following reasons

**Adding comment for posterity reasons:**

- it is part of existing code nothing updated diff displaying due to `Label` html is removed so beneath code is moved up
- Changes regarding code improvement where `km-provider-logo` can be moved in class just for duplication will be handled in other issues e.g Code Refactoring ticket